### PR TITLE
Added warning for unexpected records when parsing coordinates in PDB files.

### LINKED ISSUE #3204: PDBParser not parsing all atoms
### Setup

I am reporting a problem with Biopython 1.77 (conda, conda-forge channel) , Python 3.8.5, and operating system as follows: 

```python
Python 3.8.5 | packaged by conda-forge | (default, Jul 31 2020, 02:39:48)
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys; print(sys.version)
3.8.5 | packaged by conda-forge | (default, Jul 31 2020, 02:39:48)
[GCC 7.5.0]
>>> import platform; print(platform.python_implementation()); print(platform.platform())
CPython
Linux-4.15.0-76-generic-x86_64-with-glibc2.10
>>> import Bio; print(Bio.__version__)
1.77
```

### Expected behaviour

Parsing all atoms in the PDB file into the Biopython `structure` object. 
  
### Actual behaviour

The PDB file parses into a `structure` object however there are atoms missing. For reference, the PDB file has been modified from the one in the PDB by  [RING](http://old.protein.bio.unipd.it/ring/about) software. 

I get the warnings listed below when I parse the structure but there's nothing to suggest the missing atoms. I've parsed other PDB files and know there is normally an "atom missing" warning that can come up so I'm confused what's happening with this structure to parse without atom warnings and yet be missing atoms. 

```
/home/ubuntu/miniconda3/envs/msci_vep/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain A is discontinuous at line 24627.
  warnings.warn(
/home/ubuntu/miniconda3/envs/msci_vep/lib/python3.8/site-packages/Bio/PDB/StructureBuilder.py:89: PDBConstructionWarning: WARNING: Chain B is discontinuous at line 24745.
  warnings.warn(
```

To work out how many atoms are missing I saved the Biopython `structure` as a pdb file, assuming all atoms not in the `structure` aren't being read/can't be accessed using Biopython. 

Doing a `wc -l *` of both files shows there's a difference of 137 lines. 

> 24882 1EWQ_RING.pdb
> 24745 1EWQ_RING_bioout.pdb 

I also wrote [this script](https://github.com/edarl/rand-scripts/blob/master/missingHanalysis.py) to output exactly which atoms are missing, but I can't see anything in the atom list that looks like it wouldn't parse. 

I've attached my input file (1EWQ_RING.pdb), Biopython produced output file (1EWQ_RING_bioout.pdb) and the atom difference list (1EWQ_pdb_diff.csv) I made. 

Any help as to why those atoms aren't being parsed into Biopython would be greatly appreciated!

```python
def PDB_parser(PDB_name, PDB_path):
    parser = PDBParser(QUIET=False, PERMISSIVE=True)
    structure = parser.get_structure(PDB_name, PDB_path)
    return structure

def save_struct(structure, name):
    io = PDBIO()
    io.set_structure(structure)
    try:
        io.save(name)
        print(f"Successfully saved file:   {name}")
    except IOError:
        print("Unable to save file")
```
### Steps to reproduce
Parse PDB file to structure, save structure to a new PDB file, compare atoms in both PDB files.

FILES: [1EWQ.zip](https://github.com/biopython/biopython/files/5061544/1EWQ.zip)

### PR DESCRIPTION
**Pull request by JoaoRodrigues**
- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Resolves #3204

### CODE REVIEW

File: Bio/PDB/PDBIO.py [hunk 1]
```diff
@@ -378,6 +378,6 @@ def save(self, file, select=_select, write_end=True, preserve_atom_numbering=Fal
             if model_flag and model_residues_written:
                 fp.write("ENDMDL\n")
         if write_end:
-            fp.write("END\n")
+            fp.write("END   \n")
```
Comment (peterjc) [inline Bio/PDB/PDBIO.py:381]: Is this what the specification says, or just to avoid a warning from the new parser?

File: Bio/PDB/PDBParser.py [hunk 2]
```diff
@@ -320,7 +329,7 @@ def _parse_coordinates(self, coords_trailer):
                         line[63:70],
                     )
                 ]
-                # U's are scaled by 10^4
+                # U"s are scaled by 10^4
```
Comment (peterjc) [inline Bio/PDB/PDBParser.py:332]: revert

File: Bio/PDB/PDBParser.py [hunk 3]
```diff
@@ -357,7 +366,7 @@ def _parse_coordinates(self, coords_trailer):
                         line[63:70],
                     )
                 ]
-                # U sigma's are scaled by 10^4
+                # U sigma"s are scaled by 10^4
```
Comment (peterjc) [inline Bio/PDB/PDBParser.py:369]: revert

File: CONTRIB.rst [hunk 4]
```diff
@@ -157,6 +156,7 @@ please open an issue on GitHub or mention it on the mailing list.
 - Jose Blanca <https://github.com/JoseBlanca>
 - Joshua Meyers <https://github.com/JoshuaMeyers>
 - João D Ferreira <https://github.com/jdferreira>
+- João Rodrigues <https://github.com/joaorodrigues>
```
Comment (peterjc) [inline CONTRIB.rst:159]: Off topic, but would changing the sorting rule to UTF-8 work better for the current set of international names? See head of file and style check via ``.travis-tox.ini`` doing:

```
$ grep "^- " CONTRIB.rst | LC_ALL=C sort -u -c -f
```

**Comment by peterjc** [comment]
Tip for next time - running black will fix the quotes for you.

**Comment by JoaoRodrigues** [comment]
Well, motivation to get the `pre-commit` stuff done :)

File: Bio/PDB/PDBIO.py (hunk 1 above)
Comment (JoaoRodrigues) [reply Bio/PDB/PDBIO.py:381]: [It's what the specification says](https://www.wwpdb.org/documentation/file-format-content/format33/sect11.html#END):
```
1 -  6        Record name    "END   "
```

File: CONTRIB.rst (hunk 4 above)
Comment (JoaoRodrigues) [reply CONTRIB.rst:159]: Yeah, I found that weird too but it was a minor thing. Changing the sorting to `grep "^- " CONTRIB.rst | LC_ALL=en_US.UTF-8 sort -u -f -c` seems to order things correcty, but would this be available everywhere?

Comment (peterjc) [reply CONTRIB.rst:159]: I don't know - but as long as it works on our main developer's machines and TravisCI, that might be OK?

File: Bio/PDB/PDBIO.py (hunk 1 above)
Comment (peterjc) [reply Bio/PDB/PDBIO.py:381]: Perfect. Thanks for confirming.

**Comment by JoaoRodrigues** [comment]
PS. Also off-topic, apparently if you write 'Resolves #...' at the end of the PR message Github auto-links the issue to the PR and closes it when the PR is merged. Seems like a tiny change to the PR template that could help automate things?

**Comment by peterjc** [comment]
Good idea on tweaking the template

File: CONTRIB.rst (hunk 4 above)
Comment (JoaoRodrigues) [reply CONTRIB.rst:159]: I guess so - personally, I got used to not using the ~ in code as it gives too much trouble! I'll open an issue and tag some of the main devs to ask.

**Pull request merged**